src/main/java/com/palantir/docker/compose/DockerComposeRule.java

```diff
@@ -94,6 +108,11 @@ protected boolean skipShutdown() {
     }
 
     @Value.Default
+    protected boolean removeConflictingContainersOnStartup() {
+        return false;
```

Comment:
I don't have a strong opinion either way, but is true a better default? As the intention of this is for testing I'm not sure there are many cases where you'd want to fail when conflicting containers exist.